src/keymap.cpp

```diff
@@ -492,6 +492,13 @@ static const std::vector<OpDesc> opdescs = {
 	{OP_INT_SET, "set", "internal-set", "", KM_INTERNAL},
 
 	{OP_INT_GOTO_URL, "gotourl", "internal-goto-url", "", KM_INTERNAL},
+	{
+		OP_ARTICLEFEED,
+		"article-feed",
+		"",
+		translatable("Go to the feed of the article"),
```

Comment:
Please add the same text to doc/keycmds.dsv, so it appears in the documentation.